CHANGELOG.md

```diff
@@ -55,6 +55,12 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
   [Vincent Isambart](https://github.com/vincentisambart)
   [#3161](https://github.com/CocoaPods/CocoaPods/issues/3161)
 
+* Added a fix for a bug where some Pods containing Swift frameworks were not
+  being written to the expected directory. Alamofire seems to be a victim in many
```

Comment:
Thanks for the info. I would like to understand why 3550 doesn't resolve it. Can you tell me where I can look to get more information?